README.md

```diff
@@ -70,6 +70,7 @@ seconds.
             hosts: []
             origin_regex: false
             forced_allow_origin_value: ~
+            skip_same_as_origin: true
```

Comment:
```suggestion
            force_for_same_origin: true
```

I feel `skip` is not the best fit here, as this option is really "skipping the skip", so to say..

Reply:
With the wording change shouldn't the default be `false` then?
The current and default behavior is to not force the sending of headers if same as origin (ie: early return, skip).

Reply:
IMO the name is fine. It's not really skipping skip, it's skipping the same as origin check or it is not skipping it.